src/panel_gwalker/_gwalker.js

```diff
@@ -19,17 +19,40 @@ function cleanToDict(value){
     return value
 }
 
+function generateRandomId(length = 8) {
```

Comment:
Why not use `crypto.randomUUID()`?

Reply:
Wasn't aware it was as browser compatible as it is.